src/feedback/AlertList.jsx

```diff
@@ -17,11 +31,15 @@ class AlertList extends Component {
       const {
         id, code, userMessage, messageType, data,
       } = message;
+      if (code != null) {
```

Comment:
This is a problem - this needs to first check whether we have a normal message for this code.  Only if we don't should we check if we have an alert message template.